pytorch_lightning/trainer/trainer.py

```diff
@@ -2014,19 +2014,26 @@ def lightning_optimizers(self) -> Dict[int, LightningOptimizer]:
         return self.strategy._lightning_optimizers
 
     @property
-    def lr_schedulers(self) -> List[LRSchedulerConfig]:
+    def lr_scheduler_configs(self) -> List[LRSchedulerConfig]:
         return self.strategy.lr_schedulers
 
-    @lr_schedulers.setter
-    def lr_schedulers(self, new_schedulers: List[LRSchedulerConfig]) -> None:
-        self.strategy.lr_schedulers = new_schedulers
+    @property
+    def lr_schedulers(self) -> List[Dict[str, Any]]:
+        rank_zero_deprecation(
+            "`Trainer.lr_schedulers` is deprecated in v1.6 and will be removed in v1.8."
+            " You can use `trainer.lr_scheduler_configs` instead which contains dataclasses instead of dictionaries.",
```

Comment:
```suggestion
            " You can use `trainer.lr_scheduler_configs` instead.",
```
it will return dictionaries only right? since you have used `asdict` down below.

Reply:
`lr_scheduler_configs` returns dataclasses
`lr_schedulers` returns dictionaries

Here I'm pointing the difference to the user.